sponsors.html

```diff
@@ -62,7 +62,49 @@
 
 		  <!-- Three columns of text below the carousel -->
 		  <div class="row">
-				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code</p>
+		  		<br />
+		  		<br />
+				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code. 
+				<br />
+				Interested in donating resources to a club? Click <a href="#" data-toggle="modal" data-target="#sponsorModal">here</a>
+
+					<div class="modal fade sponsorModal" tabindex="-1" role="dialog" aria-labelledby="sponsorModal" aria-hidden="true" id="bannerformmodal">
```

Comment:
The modal is not opening for me. Looks like your modal id should be sponsorModal.